sky/serve/service_spec.py

```diff
@@ -197,17 +219,26 @@ def max_replicas(self) -> Optional[int]:
         return self._max_replicas
 
     @property
-    def qps_upper_threshold(self) -> Optional[float]:
-        return self._qps_upper_threshold
-
-    @property
-    def qps_lower_threshold(self) -> Optional[float]:
-        return self._qps_lower_threshold
+    def target_qps_per_replica(self) -> Optional[float]:
+        return self._target_qps_per_replica
 
     @property
     def post_data(self) -> Optional[Dict[str, Any]]:
         return self._post_data
 
     @property
     def auto_restart(self) -> bool:
-        return self._auto_restart
+        # Deprecate auto_restart = False
+        return True
```

Comment:
Should we remove this property?

Reply:
Do we need to remove `auto_restart` field from every class?

Reply:
might affect db.

Reply:
Why will it affect DB? IIUC, we don't store a `service_spec` class into DB?

Reply:
`serve_state`

Reply:
Discussed in Slack.